contracts/staking/IDelegationManager.sol

```diff
@@ -4,21 +4,38 @@ pragma solidity ^0.5.2;
 contract IDelegationManager {
   event Staked(address indexed user, uint256 indexed delegatorId, uint256 indexed activatonEpoch, uint256 amount, uint256 total);
```

Comment:
Pls change event names which are colliding with StakeManager. 
On Heimdall, we search by event names irrespective of the contract from which events are triggered..

Reply:
did